pkgs/top-level/all-packages.nix

```
@@ -32284,12 +32269,8 @@ with pkgs;
 
   stt = callPackage ../tools/audio/stt { };
 
-  stuntrally = callPackage ../games/stuntrally {
-    ogre = ogre1_9;
-    mygui = mygui.override {
-      withOgre = true;
-    };
-  };
+  stuntrally = callPackage ../games/stuntrally
+    { stdenv = gcc10StdenvCompat; ogre = ogre1_9; mygui = mygui.override { withOgre = true; }; };
```

This formatting is very odd.

```suggestion
;
  stuntrally = callPackage ../games/stuntrally {
    stdenv = gcc10StdenvCompat;
    ogre = ogre1_9;
    mygui = mygui.override { withOgre = true; };
  };
```